vale/.vale.ini

```diff
@@ -13,7 +13,7 @@ txt = md
 # ignore TOC marker
 TokenIgnores = (<%.*%>), (:[a-z]*:), ({:toc}), ({:notoc}), {: .*}
 
-BlockIgnores = (?s) (<%.*%>)
+BlockIgnores = (?s) *(<%.*?%>)
```

Comment:
This change does a couple of things:

- The leading white space is now optional.
- The closing tag matching (`%>`) is no longer greedy. Roughly it means that given a string (or sequence of lines) such that `<% first_unlintable %> lintable text <% second_unlintable %>`, it will match `<% first_unlintable %>` and `<% second_unlintable %>`, not the entire string.